recipes-core/everest/everest-core_2024.6.0.bb

```diff
@@ -1,17 +1,17 @@
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
 
-SRC_URI = "git://github.com/EVerest/everest-core.git;branch=main;protocol=https \
+SRC_URI = "git://github.com/EVerest/everest-core.git;branch=release/2024.6.0;protocol=https \
```

Comment:
The stable releases (.3 .6 .9 and .12 this year) will get their own branches since they do not contain everything at that point in main. The releases in-between just use a tag on main :)